pleskdistup/actions/emails.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import json
+import subprocess
+
+from pleskdistup.common import action, log, util
+
+
+class IncreaseDovecotDHParameters(action.ActiveAction):
+    def __init__(self):
+        self.name = "increase Dovecot DH parameters size to 2048 bits"
+        self.dhparam_size = 2048
+
+    def _is_required(self) -> bool:
+        proc = subprocess.run(
+            ["/usr/sbin/plesk", "sbin", "sslmng", "--show-config"],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+        )
+        if proc.returncode != 0:
+            log.warn(f"Failed to get ssl configuration by plesk sslmng: {proc.stdout}\n{proc.stderr}")
+            return False
```

Comment:
Do we really want to skip it in case of failure?

Reply:
What about that?

Reply:
Previous commentary sucked as pending. Please take a look